crates/api/src/comment_report/create.rs

```diff
@@ -31,12 +31,7 @@ impl Perform for CreateCommentReport {
 
     // check size of report and check for whitespace
     let reason = data.reason.trim();
-    if reason.is_empty() {
-      return Err(LemmyError::from_message("report_reason_required"));
-    }
-    if reason.chars().count() > 1000 {
-      return Err(LemmyError::from_message("report_too_long"));
-    }
+    check_report_reason(reason)?;
```

Comment:
Could you also add a slur check here too? Also ones for post_report and private_message_report.